tests/main.cpp

```diff
@@ -64,7 +62,7 @@ TEST_CASE("Request Airdrop") {
   std::this_thread::sleep_for(std::chrono::seconds(15));
```

Comment:
this won't work reliably, issue is that you use getBalance with the default commitment level, confirmation is dependent on voting not on wall clock time: https://docs.solana.com/cluster/commitments

Reply:
also try airdropping a smaller amount, for the sake of the test you just need 5001 lamports